uci-pharmsci/docs/.vimrc

```diff
@@ -0,0 +1,28 @@
+set nocompatible
```

Comment:
nocompatible?

Reply:
https://askubuntu.com/questions/353911/hitting-arrow-keys-adds-characters-in-vi-editor

If you're using `vi` instead of `vim`, this stops adding characters when you try to press the arrow keys.